cmd/web/tmpl/showpaste.tmpl

```diff
@@ -1,11 +1,18 @@
 {{template "header" .}}
 
+{{if .UserInfo}}
+{{if ne .UserInfo.ProfilePicURL .PasterProfilePicURL}}
```

Comment:
It seems odd to check that the pic URLs are `ne`. It's a reasonable proxy, but might be a foot gun when we really want to check `if neq .UserID .PasterUserID`, right?

Reply:
that could be arranged, but at this point the database is already updated with the correct value. User profile data is upserted on every page load, so those two can never get out of sync